tests/test_text_generation_example.py

```diff
@@ -37,6 +37,8 @@
         ],
         "fp8": [
             ("tiiuae/falcon-180B", 52.85086442722326),
+            ("meta-llama/Llama-2-7b-hf", 0.0),
+            ("meta-llama/Llama-2-70b-hf", 0.0),
```

Comment:
Baselines are 0.0 here because we use the baselines defined in `LLAMA2_FP8_CONFIG` right?

Reply:
Yes.